website/docs/patterns/multi-select-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,63 @@
+The Helios Table [selection components](/components/table#multi-select) support the selection of one or more rows within a Table and enables the user to edit the same parameter across multiple results, creating more customized views within a Table, and bulk delete results from a data set.
+
+!!! Info
+
+We don’t publish a specific component to support this functionality, instead these guidelines are intended to outline the requirements and recommendations to compose a Multi-select pattern using HDS components and should serve as a starting point. Not all scenarios are covered, if straying from the pattern in more complex scenarios, try to leverage the same core concepts and communication methods.
+!!!
+
+## Selected count
+
+Display the **selected count** to communicate how many rows or results have been selected in the _entire_ data set. This includes rows that are outside of the current view in the Table (e.g., in a different page if using [Pagination](/components/pagination)), and should be represented using a numerical value.
+
+![Example of the selected count](/assets/patterns/multi-select-patterns/selected-count.png)
+
+Pair this with a **message** to clearly communicate that a selection has been made. This can be a simple generic message stating "{number in current selection} selected", or can be more specific to the type of object in the data set (e.g., a user, cluster, workspace, etc).
+
+![Generic and specific messaging](/assets/patterns/multi-select-patterns/types-of-messages.png)
+
+## Total count
+
+If technically feasible, communicate the selected count as a portion of the **total count**, or total number of results in the Table. This can give the user a better overall snapshot of what they have selected and how the selection compares to the entire data set. This can also more effectively communicate that there may be results that are selected across multiple pages.
+
+![Total count of items and count of selected items](/assets/patterns/multi-select-patterns/total-count-selected-count.png)
+
+!!! Info
+
+Depending on how the data within a Table is fetched or if [cursor-based pagination](/components/pagination#numbered-vs-compact) is used, it may not be possible to get the total number of results in a data set.
+!!!
+
+## Bulk actions
+
+The act of selecting multiple results in a Table implies that the the user wants to edit, remove, or transform these results in some way. Consolidate these actions in a [List](/components/dropdown#list) toggled by a `small` [Dropdown](/components/dropdown) inline with the selected count, message, and total count.
+
+![Example of bulk actions](/assets/patterns/multi-select-patterns/bulk-actions.png)
+
+When working with instances of a Table where only a single action is necessary (e.g., bulk deleting a selection), use a `small` [Button](/components/button) in place of the Dropdown. We recommend using only `secondary` or `critical` Buttons in this scenario depending on the intent of the action.
```

Comment:
For Table instances requiring a single action (like bulk deletion), use a `small` [Button](/components/button) instead of a Dropdown. We advise using either `secondary` or `critical` Buttons, based on the action’s intent.